config.json

```diff
@@ -1,836 +1,793 @@
 {
-  "language": "Haskell",
   "active": true,
-  "solution_pattern": "example.*[.]hs",
-  "ignore_pattern": "example",
   "exercises": [
     {
-      "uuid": "c97d9dda-c7ee-4595-b447-693559d03ba5",
-      "slug": "hello-world",
       "core": false,
-      "unlocked_by": null,
       "difficulty": 1,
-      "topics": [
-      ]
+      "slug": "hello-world",
+      "topics": null,
+      "unlocked_by": null,
```

Comment:
shouldn't it be the case that unlocked_by and core are next to each other? They seem too closely related to separate in this manner.